webpack.common.js

```diff
@@ -10,6 +10,8 @@ if ( process.env.CIRCLECI ) {
 	workerCount = Math.max( 2, Math.floor( require( 'os' ).cpus().length / 2 ) );
 }
 
+workerCount = Math.max( 1, Math.floor( workerCount / 2 ) );
```

Comment:
Should this take into account the type of build that's taking place? Only production runs two builds concurrently. Will this not cut the number of workers in half for dev build too?

Reply:
> We could add another variable to disable this in the case that we know we're running exactly one build concurrently

Maybe we should add `CONCURRENT_BUILDS=2` to our scripts that build concurrently, then we can use that env var as the divisor for workers?

Reply:
Oops, sorry, missed your comment on the description there.

Yes, that sounds like a great idea! 👍

Reply:
I applied this suggestion and added the concurrent build flag to the relevant script.

Used this patch for some quick testing to compare with master:

```patch
diff --git a/webpack.common.js b/webpack.common.js
index dd361efda7..3e5db8f1cf 100644
--- a/webpack.common.js
+++ b/webpack.common.js
@@ -39,3 +39,5 @@ if ( concurrentBuilds > 1 ) {
 module.exports = {
 	workerCount,
 };
+
+console.log( '📣📣📣📣 I can access -> %o <- builds!📣📣📣📣', workerCount );
```
- This branch
  - `npm run build-client` reports `8`
  - `npm run build-client-both` reports `4` for evergreen and fallback builds
```
📣📣📣📣 I can access -> 8 <- builds!📣📣📣📣

[fallback ] 📣📣📣📣 I can access -> 4 <- builds!📣📣📣📣
[evergreen] 📣📣📣📣 I can access -> 4 <- builds!📣📣📣📣
```


- `master`
  - `npm run build-client` reports `8`
  - `npm run build-client-both` reports `8` for evergreen and fallback builds

```
📣📣📣📣 I can access -> 8 <- builds!📣📣📣📣
[fallback ] 📣📣📣📣 I can access -> 8 <- builds!📣📣📣📣
[evergreen] 📣📣📣📣 I can access -> 8 <- builds!📣📣📣📣
```

Seems like just what we need 🙂